frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -324,13 +324,6 @@ $monitoring-line-height: 18px;
   }
 }
 
-.query-browser__pagination {
```

Comment:
Looks like this is still used here: https://github.com/openshift/console/blob/master/frontend/public/components/monitoring/metrics.tsx#L598

Reply:
Removed since it looks to be no longer needed.